src/ServiceControl.TransportAdapter/FailedMessageForwarder.cs

```diff
@@ -71,11 +84,26 @@ static Task Forward(MessageContext context, IDispatchMessages forwarder, string
 
         public async Task Stop()
         {
-            var stoppedFronEnd = await frontEnd.StopReceiving().ConfigureAwait(false);
-            var stoppedBackEnd = await backEnd.StopReceiving().ConfigureAwait(false);
+            IStoppableRawEnedpoint stoppedFronEnd = null;
```

Comment:
Not sure we can do anything about it here but `IStoppableRawEnedpoint` has a typo in it.